Audit log empty after upgrading to 13.0

**Describe the bug**
After upgrading to PF13.0 I noticed that the radius auditing section is not recording any entry

**To Reproduce**
Steps to reproduce the behavior:
1. Upgrade to PF13
2. Perform a bunch of 802.1x/portal authentications
3. Go to "auditing"
4. No records are present


**Expected behavior**
Radius audit logs should be visible

**Additional context**
In /usr/local/pf/logs/pfcron.log I see
```
[root@srvpf ~]# grep flush_radius_audit_log /usr/local/pf/logs/pfcron.log

Oct  9 16:55:21 srvpf pfcron-docker-wrapper[3694]: t=2023-10-09T16:55:21+0200 lvl=info msg="task 'flush_radius_audit_log' created with id 18 with schedule of @every 1m" pid=7
Oct  9 16:56:21 srvpf pfcron-docker-wrapper[3694]: t=2023-10-09T16:56:21+0200 lvl=info msg="Running flush_radius_audit_log" pid=7
Oct  9 16:56:21 srvpf pfcron-docker-wrapper[3694]: t=2023-10-09T16:56:21+0200 lvl=eror msg="Job flush_radius_audit_log panic: runtime error: index out of range [1] with length 0" pid=7
Oct  9 16:57:21 srvpf pfcron-docker-wrapper[3694]: t=2023-10-09T16:57:21+0200 lvl=info msg=" Skipped flush_radius_audit_log" pid=7
Oct  9 16:58:21 srvpf pfcron-docker-wrapper[3694]: t=2023-10-09T16:58:21+0200 lvl=info msg=" Skipped flush_radius_audit_log" pid=7
Oct  9 16:59:21 srvpf pfcron-docker-wrapper[3694]: t=2023-10-09T16:59:21+0200 lvl=info msg=" Skipped flush_radius_audit_log" pid=7
Oct  9 17:00:21 srvpf pfcron-docker-wrapper[3694]: t=2023-10-09T17:00:21+0200 lvl=info msg=" Skipped flush_radius_audit_log" pid=7
Oct  9 17:01:21 srvpf pfcron-docker-wrapper[3694]: t=2023-10-09T17:01:21+0200 lvl=info msg=" Skipped flush_radius_audit_log" pid=7
Oct  9 17:02:21 srvpf pfcron-docker-wrapper[3694]: t=2023-10-09T17:02:21+0200 lvl=info msg=" Skipped flush_radius_audit_log" pid=7
Oct  9 17:03:21 srvpf pfcron-docker-wrapper[3694]: t=2023-10-09T17:03:21+0200 lvl=info msg=" Skipped flush_radius_audit_log" pid=7
```

If I try to run "/usr/local/pf/bin/pfcmd pfcron flush_radius_audit_log" manually I get the same error:
```
[root@srvpf ~]# /usr/local/pf/bin/pfcmd pfcron flush_radius_audit_log
panic: runtime error: index out of range [1] with length 0

goroutine 1 [running]:
github.com/inverse-inc/packetfence/go/cron.(*FlushRadiusAuditLogJob).argsFromEntry(0xc0002440a0?, {0x0, 0x0, 0xc000244050?})
        /root/rpmbuild/centos-8/BUILD/packetfence-13.0.0/go/cron/flush_radius_audit_log_job.go:167 +0x1058
github.com/inverse-inc/packetfence/go/cron.(*FlushRadiusAuditLogJob).buildQuery(0xc0000c5b38?, {0xc00023a000, 0x64, 0x0?})
        /root/rpmbuild/centos-8/BUILD/packetfence-13.0.0/go/cron/flush_radius_audit_log_job.go:158 +0x1e5
github.com/inverse-inc/packetfence/go/cron.(*FlushRadiusAuditLogJob).flushLogs(0xc00023c000?, {0xc00023a000?, 0x6000?, 0x6feaa0?})
        /root/rpmbuild/centos-8/BUILD/packetfence-13.0.0/go/cron/flush_radius_audit_log_job.go:75 +0x3f
github.com/inverse-inc/packetfence/go/cron.(*FlushRadiusAuditLogJob).Run(0xc0001b6150)
        /root/rpmbuild/centos-8/BUILD/packetfence-13.0.0/go/cron/flush_radius_audit_log_job.go:63 +0x22f
main.runJobNow({0x7ffcc9f475bd, 0x16}, 0x5f82b0a108?)
        /root/rpmbuild/centos-8/BUILD/packetfence-13.0.0/go/cmd/pfcron/main.go:108 +0x304
main.main()
        /root/rpmbuild/centos-8/BUILD/packetfence-13.0.0/go/cmd/pfcron/main.go:153 +0x1c5

```

In another bug report I found the command "redis-cli --raw LRANGE RADIUS_AUDIT_LOG 0 -1", if I run it I can see all of the missing radius logs

--- a/go/cron/flush_radius_audit_log_job.go
+++ b/go/cron/flush_radius_audit_log_job.go
@@ -2,6 +2,7 @@ package maint
 
 import (
 	"context"
+	"encoding/base64"
 	"encoding/json"
 	"fmt"
 	"sort"
@@ -60,7 +61,17 @@ func (j *FlushRadiusAuditLogJob) Run() {
 
 		var entries [][]interface{} = make([][]interface{}, 0, len(a))
 		for _, jsonStr := range a {
-			var entry []interface{} = make([]interface{}, 3)
+			var entry []interface{} = make([]interface{}, 4)
+			if jsonStr[0] != '[' {
+				s, err := base64.StdEncoding.DecodeString(jsonStr)
+				if err != nil {
+					log.LogError(ctx, fmt.Sprintf("%s error running: %s", j.Name(), err.Error()))
+					continue
+				}
+
+				jsonStr = string(s)
+			}
+
 			err := json.Unmarshal([]byte(jsonStr), &entry)
 			if err != nil {
 				log.LogError(ctx, fmt.Sprintf("%s error running: %s", j.Name(), err.Error()))

--- a/go/cron/flush_radius_audit_log_job_test.go
+++ b/go/cron/flush_radius_audit_log_job_test.go
@@ -2,6 +2,7 @@ package maint
 
 import (
 	"context"
+	"encoding/base64"
 	"encoding/json"
 	"testing"
 )
@@ -81,7 +82,7 @@ func TestFlushRadiusAuditLogFromRedis(t *testing.T) {
 	}
 	redis := redisClient()
 	redis.Del(ctx, "RADIUS_AUDIT_LOG")
-	redis.LPush(ctx, "RADIUS_AUDIT_LOG", RADIUS_ENTRY, RADIUS_ENTRY)
+	redis.LPush(ctx, "RADIUS_AUDIT_LOG", RADIUS_ENTRY, base64.StdEncoding.EncodeToString([]byte(RADIUS_ENTRY)))
 
 	res, err := db.Exec("DELETE FROM radius_audit_log;")
 	if err != nil {

--- a/raddb/policy.d/packetfence
+++ b/raddb/policy.d/packetfence
@@ -89,7 +89,7 @@ packetfence-audit-log-accept {
     }
     else {
         request-timing
-        %{redis:RPUSH RADIUS_AUDIT_LOG '["Accept",%{json_encode:&request:[*]},%{json_encode:&reply:[*]},%{json_encode:control:PacketFence-Computer-Name control:PacketFence-Switch-Id control:PacketFence-Switch-Mac control:PacketFence-Switch-Ip-Address control:PacketFence-IfIndex control:PacketFence-Connection-Type control:Auth-Type control:PacketFence-Role control:PacketFence-Status control:PacketFence-Profile control:PacketFence-Source control:PacketFence-AutoReg control:PacketFence-IsPhone control:PacketFence-Request-Time Packet-Src-IP-Address}]'}
+        %{redis:RPUSH RADIUS_AUDIT_LOG '%{base64:["Accept",%{json_encode:&request:[*]},%{json_encode:&reply:[*]},%{json_encode:control:PacketFence-Computer-Name control:PacketFence-Switch-Id control:PacketFence-Switch-Mac control:PacketFence-Switch-Ip-Address control:PacketFence-IfIndex control:PacketFence-Connection-Type control:Auth-Type control:PacketFence-Role control:PacketFence-Status control:PacketFence-Profile control:PacketFence-Source control:PacketFence-AutoReg control:PacketFence-IsPhone control:PacketFence-Request-Time Packet-Src-IP-Address}]}'}
 #        %{redis:LPUSH ACCEPT_RADIUS_AUDIT_LOG 1}
 #       -sql {
 #           fail = 1
@@ -103,7 +103,7 @@ packetfence-audit-log-reject {
     }
     else {
         request-timing
-        %{redis:RPUSH RADIUS_AUDIT_LOG '["Reject",%{json_encode:&request:[*]},%{json_encode:&reply:[*]},%{json_encode:control:PacketFence-Computer-Name control:PacketFence-Switch-Id control:PacketFence-Switch-Mac control:PacketFence-Switch-Ip-Address control:PacketFence-IfIndex control:PacketFence-Connection-Type control:Auth-Type control:PacketFence-Role control:PacketFence-Status control:PacketFence-Profile control:PacketFence-Source control:PacketFence-AutoReg control:PacketFence-IsPhone control:PacketFence-Request-Time Packet-Src-IP-Address}]'}
+        %{redis:RPUSH RADIUS_AUDIT_LOG '%{base64:["Reject",%{json_encode:&request:[*]},%{json_encode:&reply:[*]},%{json_encode:control:PacketFence-Computer-Name control:PacketFence-Switch-Id control:PacketFence-Switch-Mac control:PacketFence-Switch-Ip-Address control:PacketFence-IfIndex control:PacketFence-Connection-Type control:Auth-Type control:PacketFence-Role control:PacketFence-Status control:PacketFence-Profile control:PacketFence-Source control:PacketFence-AutoReg control:PacketFence-IsPhone control:PacketFence-Request-Time Packet-Src-IP-Address}]}'}
 #       -sql_reject {
 #           fail = 1
 #       }

